src/test/logical_query_plan/stored_table_node_test.cpp

```diff
@@ -156,7 +156,7 @@ TEST_F(StoredTableNodeTest, FunctionalDependenciesNone) {
   table->add_soft_unique_constraint({_a->original_column_id, _b->original_column_id, _c->original_column_id},
                                     IsPrimaryKey::No);
 
-  EXPECT_EQ(_stored_table_node->functional_dependencies().size(), 0);
+  EXPECT_TRUE(_stored_table_node->functional_dependencies().empty());
```

Comment:
I decided to use `EXPECT_EQ` for better console output (all values instead of simple fail message). 
No issue though.

Reply:
What do you mean with "all values"? Does that actually print the FDs?

Reply:
No, but in case the container is not empty, `EXPECT_EQ` outputs the container's size.